### ISSUE TITLE
Runtime died, Restarting automatically

### ISSUE THREAD

**Issue by ahmedshahin9** [googlecolab/colabtools#38]
I have a problem, when I train unet network using the attached code, it stops and prompts "Runtime died, restarting"
And I cannot train the network properly
kindly help me

**Comment by furkanyildiz** [comment]
Did you solve the problem?

**Comment by ahmedshahin9** [comment]
yes I guess, it is a bug in my code,  the layers parameter which i sit to 23 is very large number of layers so the memory got filled and the system restarts

**Comment by saksham7778** [comment]
can u help me in my code

**Comment by ahmedshahin9** [comment]
@saksham7778 Sure, I think it happened due to a memory issue.
